packages/web-components/src/components/va-search-input/va-search-input.tsx

```diff
@@ -120,7 +136,14 @@ export class VaSearchInput {
    * Fires a submit and analytics events.
    */
   private handleSubmit = () => {
-    this.inputRef.dispatchEvent(
+    const form = this.el.shadowRoot.querySelector('form');
+    if (form) {
+      form.onsubmit = (event) => {
+        event.preventDefault();
```

Comment:
The `form` element will submit (reloading the page) unless we prevent the default behavior.